README.md

```diff
@@ -1,76 +1,61 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Covid-Data Metrics
 
-> One paragraph statement about the project.
+> 
 
-![screenshot](./app_screenshot.png)
+![screenshot](./images/screenshot.png)
```

Comment:
- [ ] Link to this screenshot is broken. Please let's fix it. You may consider doing something like `![screenshot](./src/images/screenshot.PNG)`